Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -4574,6 +4572,9 @@ void HTMLMediaElement::setControls(bool controls)
 
 double HTMLMediaElement::volume() const
 {
+    if (m_implicitlyMuted.value_or(false))
```

Comment:
We do this pattern a lot: `m_implicitlyMuted.value_or(false)`. Could we just add a `bool implicitlyMuted() const { return m_implicitlyMuted.value_or(false); }` to HTMLMediaElement?